### DISCUSSION TITLE
Format field inconsistencies

### DISCUSSION THREAD

**Discussion by adgad** [Financial-Times/content-tree#24] [Team Posts]
- in-article images have `standardInline`, `desktop`, `mobile`
- "main" image at top of body (so still technically inline) has `standard` but not `standardInline`
- scrollytelling images only have `desktop` and `mobile`, although they actually mean `landscape` and `portrait` - but no `standardInline`
- `ftEditSquareImage` is a format?

More generally - what is the meaning of this field (if any?). It is kind of conflating a few different things (the crop of the image, the viewport size, the product displaying it). Is that okay / what we want? If so, how do we ensure that the list of possible formats makes sense and doesn't spiral out of control.

**Comment by atanasdinov** [comment]
We're fine with introducing an enum validation for these and / or supporting additional "formatting" fields. We'd have to make sure to update old content if we choose the latter though.
